src/index.ts

```diff
@@ -18,7 +18,6 @@ export * as number from './utils/number';
 export * as transaction from './utils/transaction';
 export * as stark from './utils/stark';
 export * as merkle from './utils/merkle';
-export * as ec from './utils/ellipticCurve';
```

Comment:
It might be worthwhile to reinstate the `ec` utility export as an alias to the corresponding `@noble/curves` methods.

Probably best not to do it in this PR, just something to keep in mind.